bin/configs/csharp-generichost-tags-latest.yaml

```diff
@@ -0,0 +1,10 @@
+# for csharp generichost
+generatorName: csharp
+outputDir: samples/client/petstore/csharp/generichost/net8/Tags
```

Comment:
shall we add the new folder to the github workflow?

Reply:
Yes, I missed committing it.